Resources/Prototypes/Entities/Structures/Piping/Atmospherics/pipes.yml

```diff
@@ -1,39 +1,14 @@
 ﻿- type: entity
   abstract: true
   id: GasPipeBase
+  parent: BaseItem
```

Comment:
this is sus but mgiht work fine

Reply:
Unless BaseItem gets changed, then it just had redundant components, so I consolidated them. If it's contentious, then I can remove it from this PR.

Reply:
should be alright